sdk-src/src/main/java/com/hpe/adm/nga/sdk/EntityListService.java

```diff
@@ -365,6 +366,23 @@ EntityModel getEntityModel(JSONObject jsonEntityObj) {
         return entityModel;
     }
 
+    private Date getDateInLocalTimeZoneFromUTC(Date date){
+        //Get the time info and add the server timezone
+        TimeZone serverTimeZone = TimeZone.getTimeZone("UTC");
+        SimpleDateFormat formatter = new SimpleDateFormat("dd MMM yyyy HH:mm:ss");
+        String strDateString = formatter.format(date);
+        strDateString+=" "+serverTimeZone.getID();
+        //change to local time zone
+        formatter.setTimeZone(TimeZone.getDefault());
+        formatter.applyPattern("dd MMM yyyy HH:mm:ss z");
```

Comment:
Yes, in my approach to the problem the date from the server has to be changed in UTC to be able to switch it to the  local time. 
The problem is that the server is giving me the UTC, but in other format (it is called ISO8601. It adds a Z at the end of the date. Z ="zero hour offset" , "Zulu time"). the SimpleDateFormat does not care about this. (It changes the time zone to my local time zone but it does not change the time. If I get from server 8:30 Z it changes it to 8:30 EET).
I was not able to find a solution that uses the server time directly. Do you have any ideas ?